managed/src/main/java/com/yugabyte/yw/scheduler/Scheduler.java

```diff
@@ -184,10 +184,10 @@ else if (cronExpression != null) {
             this.runMultiTableBackupsTask(schedule);
           }
         }
-        List<Backup> expiredBackups = Backup.getExpiredBackups(schedule.scheduleUUID);
-        for (Backup backup : expiredBackups) {
-          this.runDeleteBackupTask(backup, schedule);
-        }
+      }
+      List<Backup> expiredBackups = Backup.getExpiredBackups();
```

Comment:
We need to not run this per schedule now. So you can move this outside the scope of the for loop of schedules.

Reply:
It is outside of the `for` loop. Guess you got confuse with `try` block `}`